src/checkers/inference/InferenceOptions.java

```diff
@@ -122,6 +122,9 @@
     @Option("For inference, add debug on the port indicated")
     public static String debug;
 
+    @Option("If ture, don't insert solutions that are equivalent to the default ones back to the code.")
+    public static boolean ignoreDefaultAnnotations = true;
```

Comment:
I didn't find the `ignore` particularly intuitive to understand what this option does. `donotInsertDefaultAnnotations` is too long. Can you think of something better?

Reply:
How about `keepDefaultsImplicit`?